src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/LocationMapper.kt

```diff
@@ -11,4 +11,8 @@ class LocationMapper {
   fun mapToCurrentRequestBasePath(path: String, id: UUID): URI {
     return ServletUriComponentsBuilder.fromCurrentRequest().path(path).buildAndExpand(id).toUri()
   }
+
+  fun mapToCurrentContextPathAsString(path: String, id: UUID): String {
```

Comment:
i appreciate this interface is motivated by who is calling this method, but it looks slightly odd to me that the two methods in this class have different return types.

Reply:
Made consistent